src/cli/ui/ui.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
```

Comment:
Doesn't necessarily need to be resolved but I'm more used to seeing `/components` instead of `/ui`.

Reply:
Planning to break this into other components, i.e. the UI folder can have a folder `components` and then we can import using 

```ts
import {Component} from './component/Component'
``` 

Thoughts?

Reply:
What else will go under `/ui`? I guess I just don't see the benefit of having a `/ui` folder if it doesn't contain anything but another folder! In the end it's not the end of the world. :)

Reply:
Yeah fair, I was hoping to get a test file viewer in, but that is probably a stretch goal too, for now this should be fine